index.html

```diff
@@ -180,5 +181,47 @@ <h3>Skills</h3>
         </div>
       </section>
     </section>
+
+    <footer class="form-section">
+      <h2 class="form-desc">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+
+      <form action="https://formspree.io/f/xbjwrppl" method="POST" id="my-form">
+        <input
+          type="text"
+          id="name"
+          name="user_name"
+          placeholder="Full Name"
+          maxlength="30"
+          required
+        />
+
+        <input
+          type="email"
+          id="email"
+          placeholder="Email address"
+          name="user_email"
+          required
+        />
+
+        <textarea
+          name="message"
+          id="msg"
+          placeholder="Enter text here"
+          required
+        ></textarea>
```

Comment:
- [ ] Textarea should have `maxlength="500"` attribute. Please kindly fix this issue and form fill be completed 🙌😊